my_game.py

```diff
@@ -80,8 +83,11 @@ def __init__(self):
             scale=SPRITE_SCALING
         )
 
+        self.angle = random.randint(-90, 180)
```

Comment:
Why is it -90 to 180?? 1-360 would be what you'd expect. Anyway. turns out `Arcade` has a function for this. Please use it: https://api.arcade.academy/en/latest/api/utility.html#arcade.rand_angle_360_deg